### DISCUSSION TITLE
Multiple await arg() in script possible?

### DISCUSSION THREAD

**Discussion by nielsbom** [johnlindquist/kit#239] [Q&A]
Is it possible to have multiple `await arg()` statements in a script?

When I run this code the second prompt for input stays open, which is not what I want.

```javascript
const input1 = await arg("q1");
const input2 = await arg("q2");
notify("bla");
```

Or is that a bug?

**Comment by johnlindquist** [comment]
`notify` is based on [node-notifier](https://www.npmjs.com/package/node-notifier)

```js
const title = await arg("title")
const message = await arg("message")
notify({ title, message })
```

The issue is that notifier has event listeners that hang around since it's possible to click/interact with the notification and get data back.

For now, you can use the following to hide the prompt immediately:
```js
const title = await arg("title")
const message = await arg("message")
notify({ title, message })
send("HIDE_APP")
```

I'll keep thinking about the notify API since I imagine most people won't care about using event listeners.
